### PR TITLE
Add message on version for managed backups setting + missing frequency option

### PR DESCRIPTION
**Pull request by kathancox**
Fixes DOC-11211

### CODE REVIEW

File: src/current/cockroachcloud/managed-backups.md [hunk 1]
```diff
@@ -34,6 +34,10 @@ Refer to the [Cloud Console](#restore-a-basic-cluster) section for details on vi
 
 ### Standard and Advanced clusters
 
+{{site.data.alerts.callout_info}}
```
Comment (kathancox) [inline src/current/cockroachcloud/managed-backups.md:37]: I wonder if this is necessary, given it is supported in all "supported" versions of CRDB https://www.cockroachlabs.com/docs/releases/release-support-policy#supported-versions?

Comment (alicia-l2) [reply src/current/cockroachcloud/managed-backups.md:37]: could we just call out that its in all supported versions?

**Review by alicia-l2 — COMMENTED** [review]
lgtm. I agree with you, we could just call out that its in all supported versions as opposed to saying the version itself.

File: src/current/cockroachcloud/managed-backups.md [hunk 2]
```diff
@@ -34,6 +34,10 @@ Refer to the [Cloud Console](#restore-a-basic-cluster) section for details on vi
...
+Configurable managed backup settings are available in all [supported versions]({% link releases/release-support-policy.md %}) of CockroachDB on {{ site.data.products.standard }} and {{ site.data.products.advanced }} clusters.
```
Comment (taroface) [inline src/current/cockroachcloud/managed-backups.md:38]: Should the supported versions link include the `#supported-versions` anchor?

File: src/current/cockroachcloud/plan-your-cluster.md [hunk 3]
```diff
@@ -31,7 +31,7 @@ To achieve a rough initial estimate of compute capacity, consider:
...
+After considering the above factors, you have enough information to [create a CockroachDB {{ site.data.products.standard }} cluster]({% link cockroachcloud/create-your-cluster.md %}) using the CockroachDB {{ site.data.products.cloud }} Console, the [CockroachDB Terraform provider]({% link cockroachcloud/provision-a-cluster-with-terraform.md %}), or the [CockroachDB Cloud API]({% link cockroachcloud/cloud-api.md %}). After the cluster is deployed, you can [scale its compute capacity]({% link cockroachcloud/cluster-management.md %}#edit-cluster-capacity) by increasing or decreasing its provisioned capacity as needed.
```
Comment (taroface) [inline src/current/cockroachcloud/plan-your-cluster.md:34]:
```suggestion
After considering the preceding factors, you have enough information to [create a CockroachDB {{ site.data.products.standard }} cluster]({% link cockroachcloud/create-your-cluster.md %}) using the CockroachDB {{ site.data.products.cloud }} Console, the [CockroachDB Terraform provider]({% link cockroachcloud/provision-a-cluster-with-terraform.md %}), or the [CockroachDB {{ site.data.products.cloud }} API]({% link cockroachcloud/cloud-api.md %}). After the cluster is deployed, you can [scale its compute capacity]({% link cockroachcloud/cluster-management.md %}#edit-cluster-capacity) by increasing or decreasing its provisioned capacity as needed.
```

**Review by taroface — APPROVED** [review]
LGTM! Tiny suggestions

File: src/current/cockroachcloud/managed-backups.md (hunk 2 above)
Comment (kathancox) [inline src/current/cockroachcloud/managed-backups.md:38]:
```suggestion
Configurable managed backup settings are available in all [supported versions]({% link releases/release-support-policy.md %}#supported-versions) of CockroachDB on {{ site.data.products.standard }} and {{ site.data.products.advanced }} clusters.
```

Comment (kathancox) [reply src/current/cockroachcloud/managed-backups.md]: Yes defo, thanks for that

**Comment by kathancox** [comment]
TFTRs!

**Pull request merged**